src/prefect/client/orchestration/__init__.py

```diff
@@ -607,7 +608,7 @@ async def get_runs_in_work_queue(
             List[FlowRun]: a list of FlowRun objects read from the queue
         """
         if scheduled_before is None:
-            scheduled_before = pendulum.now("UTC")
+            scheduled_before = DateTime.now("UTC")
```

Comment:
If you have the most recent changes from `main`, you should be able to use `now` from `prefect.types._datetime`.
```suggestion
            scheduled_before = now("UTC")
```

Reply:
done!